pkg/client/pods.go

```diff
@@ -92,3 +94,7 @@ func (c *pods) Update(pod *api.Pod) (result *api.Pod, err error) {
 	err = c.r.Put().Namespace(c.ns).Resource("pods").Name(pod.Name).Body(pod).Do().Into(result)
 	return
 }
+
+func (c *pods) Bind(binding *api.Binding) error {
+	return c.r.Post().Namespace(c.ns).Resource("pods").Name(binding.Name).Suffix("binding").Body(binding).Do().Error()
```

Comment:
Maybe we should create a Subresource() routine in the client, as opposed to just using Suffix(), to reinforce the new pattern.